portfolio/src/main/webapp/script.js

```diff
@@ -79,7 +79,15 @@ function getComments() {
     for (message of messages) {
       const commentContainer = document.createElement('div');
       commentContainer.className += 'comment-container';
-      commentContainer.innerText = message.email + ': ' + message.text;
+
+      const commentPara = document.createElement('p');
+      commentPara.innerText = message.email + ': ' + message.text;
+      commentContainer.appendChild(commentPara);
+
+      const sentimentPara = document.createElement('p');
+      sentimentPara.className += 'sentiment';
+      sentimentPara.innerText = 'Vibe: ' + message.score.toFixed(2);
+      commentContainer.appendChild(sentimentPara);
```

Comment:
This is starting to get lengthy. I recommend moving this into a method that returns the commentContainer.